subsys/ipc/rpmsg_service/rpmsg_backend.c

```diff
@@ -43,7 +43,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_RPMSG_SERVICE_LOG_LEVEL);
 #define IPM_WORK_QUEUE_STACK_SIZE CONFIG_RPMSG_SERVICE_WORK_QUEUE_STACK_SIZE
 
 #if IS_ENABLED(CONFIG_COOP_ENABLED)
-#define IPM_WORK_QUEUE_PRIORITY -1
+#define IPM_WORK_QUEUE_PRIORITY K_HIGHEST_APPLICATION_THREAD_PRIO
 #else
 #define IPM_WORK_QUEUE_PRIORITY 0
 #endif
```

Comment:
Uhm, actually using `K_HIGHEST_APPLICATION_THREAD_PRIO` you can remove the whole `if` block now

Reply:
Yes, I took minimum-change approach, push in minutes.